docs/general/deployment.md

```diff
@@ -137,20 +137,16 @@ _Step 3:_ Within the root of your react-boilerplate source folder, execute the f
 
 ## Netlify
 
-Netlify is a static site host that deploys from Git repos. To deploy `react-boilerplate` from Netlify:
+Netlify is a static site host that deploys from Git repos. To deploy a `react-boilerplate` app from Netlify, you can leverage their continuous deployment feature. All you have to do is push to your repo and Netlify will [run your production build script](https://docs.netlify.com/configure-builds/get-started/).
 
-_Step 1:_ Clone `react-boilerplate` and remove the default project (`npm run clean`)
+_Step 1:_ Clone react-boilerplate and remove the default project (`npm run clean`)
 
 _Step 2:_ Create a Git repo for your project on one of the supported hosts (i.e. GitHub)
 
 _Step 3:_ Create a new project on Netlify by logging in via your Git host and selecting the repo you just made
 
-_Step 4:_ In your local repo, change the remote from `react-boilerplate`'s to your own (`git remote set-url origin https://github.com/USERNAME/REPOSITORY.git`)
+_Step 4:_ In your local repo, change the remote from react-boilerplate's to your own (i.e. `git remote set-url origin https://github.com/USERNAME/REPOSITORY.git`)
 
-_Step 5:_ Comment out the `build` line from your `.gitignore` with a text editor
+_Step 5:_ Go to the **Build & deploy** section of the project settings. For "Build command," enter `npm run build`. For "Publish directory," enter `build`.
 
-_Step 6:_ Build your project (`npm run build`)
-
-_Step 7:_ Commit and push (`git add build`, `git commit -m "first build"`, `git push origin master`
-
-_Step 8:_ Netlify detects the push and automatically redeploys your site.
+_Step 6:_ Commit and push as in Step 7 above. Netlify detects the push, builds the project, and deploys the results.
```

Comment:
Which step 7 does this refer to? For the last step, I would just say: "Update your master branch (either using PRs or by committing directly to it). Netlify will detect both and automatically redeploy your app for you."

Reply:
Whoops, typo